src/app/tests/suites/certification/Test_TC_CNET_1_3.yaml

```diff
@@ -23,111 +23,114 @@ config:
+      cluster: "Network Commissioning"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: uint16
 
     - label: "TH reads the FeatureMap from DUT"
-      verification: |
-          ./chip-tool networkcommissioning read feature-map 1 0
-
+      PICS: CNET.S.F00
+      cluster: "Network Commissioning"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 1
+          constraints:
+              type: map32
 
-          Verify on the TH Log:
+    - label: "TH reads the FeatureMap from DUT"
+      PICS: CNET.S.F01
+      cluster: "Network Commissioning"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 2
+          constraints:
+              type: map32
 
-          [1651215443.195852][3285:3290] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0031 Attribute 0x0000_FFFC DataVersion: 2584224280
-          [1651215443.195939][3285:3290] CHIP:TOO:   FeatureMap: 4
-      disabled: true
+    - label: "TH reads the FeatureMap from DUT"
+      PICS: CNET.S.F02
+      cluster: "Network Commissioning"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 4
+          constraints:
+              type: map32
 
     - label: "TH reads AttributeList from DUT"
-      verification: |
-          ./chip-tool networkcommissioning read attribute-list 1 0
-
-
-          Verify on the TH Log:
-
-          [1651215479.539148][3293:3298] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0031 Attribute 0x0000_FFFB DataVersion: 2584224280
-          [1651215479.539230][3293:3298] CHIP:TOO:   AttributeList: 13 entries
-          [1651215479.539262][3293:3298] CHIP:TOO:     [1]: 0
-          [1651215479.539288][3293:3298] CHIP:TOO:     [2]: 1
-          [1651215479.539323][3293:3298] CHIP:TOO:     [3]: 2
-          [1651215479.539348][3293:3298] CHIP:TOO:     [4]: 3
-          [1651215479.539373][3293:3298] CHIP:TOO:     [5]: 4
-          [1651215479.539397][3293:3298] CHIP:TOO:     [6]: 5
-          [1651215479.539420][3293:3298] CHIP:TOO:     [7]: 6
-          [1651215479.539444][3293:3298] CHIP:TOO:     [8]: 7
-          [1651215479.539468][3293:3298] CHIP:TOO:     [9]: 65528
-          [1651215479.539493][3293:3298] CHIP:TOO:     [10]: 65529
-          [1651215479.539517][3293:3298] CHIP:TOO:     [11]: 65531
-          [1651215479.539541][3293:3298] CHIP:TOO:     [12]: 65532
-          [1651215479.539564][3293:3298] CHIP:TOO:     [13]: 65533
-      disabled: true
+      command: "readAttribute"
+      cluster: "Network Commissioning"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0xFFFD,
+                      0xFFFC,
+                      0xFFFB,
+                      0xFFF9,
+                      0xFFF8,
+                      0x00,
```

Comment:
As a followup, we could have a read of this conditioned on `F00 | F01` that checks for `ScanMaxTimeSeconds` and `ConnectMaxTimeSeconds`, right?

Reply:
Yes, but that's not part of this test plan - 1.3 covers only the global attrs.

Reply:
oh, sorry, yeah, I misunderstood. Yes, I like that